### DISCUSSION TITLE
Criar/Alterar nomes dos STATUS das OS

### DISCUSSION THREAD

**Discussion by SuperPorte** [RamonSilva20/mapos#1619] [Q&A]
Boa tarde galera
Tem como criar novos STATUS paras as OSs e alterar os que já têm lá?
Se sim, como?
Obrigado.

**Comment by SuperPorte** [comment]
Ninguém??

**Reply by bruno-esilva** [reply]
Se for apenas para visualização, vc pode editar os arquivos abaixo, que ficam em Application/View/OS

AdicionarOS.php - Linha 59
Ex.:  
<option value="Em Garantia">Em Garantia</option>

EditarOS.php - Linha 94
Ex.: 
<option <?php if ($result->status == 'Em Garantia') {
                        echo 'selected';
                    } ?> value="Em Garantia">Em Garantia
                                                </option>

Basta copiar um dos modelos que já tem lá e substituir pelo nome que você quer. Nos exemplos acima eu adicionei a opção Em Garantia. Aí vc só substitui pelo texto que vc quer. 
Porém isso é só para visualizar. Se precisar pesquisar por status ou algo assim, aí já precisaria mexer nos controllers e models, e eu não fuçei nessa parte.

**Reply by SuperPorte** [reply]
Show de bola!!!
De quebra ainda mudei para a NovaOS salvar como ABERTA e não ORÇAMENTO.

Realmente essa opção de filtrar os STATUS novos faz falta, mas já é 70% de progresso hehehe
Valeu!!

**Comment by hepazevedo** [comment]
Boa noite, amigos
Na parte de edição de OS, percebi que se pode alterar o status dela para "Faturado" através do select, o que acaba gerando a possibilidade de faturar OS sem serviços ou produtos cadastrados.
Não entendo muito de programação, mas resolvi isso fazendo uma pequena alteração em "application\views\os\editarOs.php", linha 102, como segue:
`<?php if ($result->status == 'Faturado') {
                        echo '<option selected value="Faturado">Faturado</option>';
                    } ?>`

Fica a minha sugestão
